doc/user_manual/rom.tex

```diff
@@ -2970,6 +2970,23 @@ \subsubsection{SyntheticHistory}
 
 In addition, any number of the following TSA algorithms may be included as subnodes of this ROM:
 
+\begin{itemize}
+\item \xmlNode{PolynomialRegression} fits time-series data using a polynomial
+  function of degree one or greater.
+
+  \xmlNode{PolynomialRegression} has the following attributes:
+  \begin{itemize}
+    \item \xmlAttr{target}, \xmlDesc{comma seperated string, required field}, indicates which
+        target signals should be trained as part of this ROM using this TSA algorithm.
+  \end{itemize}
+
+  \xmlNode{PolynomialRegression} has the following subnodes:
+  \begin{itemize}
+  \item \xmlNode{degree}, \xmlDesc{integer, required field}, indicates which
+    degree of polynomial to fit to the presented data.
+  \end{itemize}
+  \end{itemize}
```

Comment:
I think this itemize can probably be merged inside the itemize below, where Fourier and ARMA are both `\item` level entries, right? I imagine that would lead to
- PolynomialRegression
  - polyreg options
- Fourier
  - fourier options
- ARMA
  - arma options